src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,11 +4,15 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
+        logger.debug("completeOrder method was called. Params: userId={}", userId);
```

Comment:
if you use logger.debug() this logg won't appear in logs

Reply:
no, actually it is correct to use `debug` level for such logs

the appearence should be configured via configuration file - usually the root level is set to `info`, but when you wan't to debug smth, it is set to `debug`